Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -110,10 +110,10 @@ void GeneralWidget::CreateWidgets()
   auto* shader_compilation_layout = new QGridLayout();
 
   const std::array<const char*, 4> modes = {{
-      QT_TR_NOOP("Synchronous"),
-      QT_TR_NOOP("Synchronous (Ubershaders)"),
-      QT_TR_NOOP("Asynchronous (Ubershaders)"),
-      QT_TR_NOOP("Asynchronous (Skip Drawing)"),
+      QT_TR_NOOP("Traditional"),
```

Comment:
I like the other names. But I think 'Traditional' is worse than even 'Synchronous'. 'Synchronous' at least tries to explain it, but 'Traditional' only is meaningful if you know what the Tradition is. If it would be changed back, I would fully approve.

Reply:
I guess reverting just that one to "Synchronous" would make sense in a way... It is the only option that compiles shaders synchronously during normal gameplay, after all. What do others think about this?

Reply:
From what we discussed in IRC, basically *everything* is synchronous. Calling one option Synchronous is not accurate, so I'm against that. Traditional works because it is a valid name: it is the traditional method of shader compilation handling in Dolphin and elsewhere.

Reply:
> From what we discussed in IRC, basically *everything* is synchronous.

That was when we were talking about slowdown caused by the GPU performing work. But the other options don't do synchronous **shader compilation**, and shader compilation is exactly what this option is about, which is why I think synchronous makes sense for this option.

Reply:
My first thought is that "Standard" would probably be nice - while it's similar to "Traditional", it is more obvious for the user that they are "normal" or "the most regular".

In an emulator like this, I can imagine people thinking "Traditional" is related to classic/retro modes, as in "looking like it used to on a real console and TV back in the day".

Reply:
"Normal" was the original suggestion, but phire pointed out that it's actually pretty abnormal from a graphics standpooint, it's just the normal for us. Hence, Traditional.

Reply:
It can make sense to an emulator programmer. But to reference assumed knowledge is not a substitute for an attempt at actual description.

Reply:
Idea: 'Wait For Compile'?